src/keystore.c

```diff
@@ -106,39 +196,73 @@ void DPS_DestroyMemoryKeyStore(DPS_MemoryKeyStore* mks)
         DPS_MemoryKeyStoreEntry* entry = mks->entries + i;
```

Comment:
could this be accessing one past the end of `mks->entries`?

Reply:
i < mks->entriesCount so I think this is safe